.github/workflows/ts.yml

```diff
@@ -44,3 +44,21 @@ jobs:
         run: "yarn build"
       - name: "run test"
         run: "yarn test"
+  stale-api-schema:
+    name: API schema is up to date
```

Comment:
Not crazy about this name. Let me know if you have any preference for other names! This job could also be rolled into the build step.